front-end/src/components/Content/Users.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import './css/users.scss';
+
+export default function Users() {
+
+  const userData = [
+    {
+      "title" : "title",
+      "userName" : "Text line",
+      "email" : "test.abc@salto.link",
+      "status" : "Active"
+    }, {
+      "title" : "title",
+      "userName" : "Text line",
+      "email" : "test.abc@salto.link",
+      "status" : "Un active"
+    }, {
+      "title" : "title",
+      "userName" : "Text line",
+      "email" : "test.abc@salto.link",
+      "status" : "Active"
+    }, {
+      "title" : "title",
+      "userName" : "Text line",
+      "email" : "test.abc@salto.link",
+      "status" : "Active"
+    }, {
+      "title" : "title",
+      "userName" : "Text line",
+      "email" : "test.abc@salto.link",
+      "status" : "Active"
+    }, {
+      "title" : "title",
+      "userName" : "Text line",
+      "email" : "test.abc@salto.link",
+      "status" : "Active"
+    }, {
+      "title" : "title",
+      "userName" : "Text line",
+      "email" : "test.abc@salto.link",
+      "status" : "Active"
+    }, {
+      "title" : "title",
+      "userName" : "Text line",
+      "email" : "test.abc@salto.link",
+      "status" : "Active"
+    }, {
+      "title" : "title",
+      "userName" : "Text line",
+      "email" : "test.abc@salto.link",
+      "status" : "Active"
+    }, {
+      "title" : "title",
+      "userName" : "Text line",
+      "email" : "test.abc@salto.link",
+      "status" : "Active"
+    }
+  ]
+
+
+    return (
+      <div id="users" className=''>
+        <table>
+          <tr>
+            <th><input type="checkbox" /></th>
+            <th>Title</th>
+            <th>User Name</th>
+            <th>Email</th>
+            <th>Status</th>
+            <th>button</th>
+          </tr>
+          {userData.map((ud) => (
+            <tr>

```

Comment:
@Taiga17 -san
Add update more about the key when we loop.
https://ja.reactjs.org/docs/lists-and-keys.html
```JS
{userData.map((item, idx) => (
  <tr key={idx}>
```